### DISCUSSION TITLE
"ERROR: canopy resistance was somehow smaller than its boundary layer resistance"

### DISCUSSION THREAD

**Discussion by adamhb** [NGEET/fates#993] [Q&A]
Hi all,

I recently got this error from FATES: 

"Combined canopy resistance was somehow smaller than its boundary layer resistance component. 

 r_sb_leaves [s/m]:    85.8899710920912     
 bc_in(s)%rb_pa(ifp) [s/m]:    85.8899710920912"

The error is raised 6 months into a bare ground spinup and comes from [here](https://github.com/NGEET/fates/blob/main/biogeophys/FatesPlantRespPhotosynthMod.F90#L883).

This is for a single point simulation with driver data that I haven't used before. The driver data is subsetted from downscaled (4km) and bias corrected CRUNCEP data. So the issue seems to have something to do with the driver data. Does anyone have any ideas for what about the new driver data could be causing this issue?

Thank in advance for any help / thoughts!
Adam

**Comment by ckoven** [comment]
Hi @adamhb, I wonder what is the relative humidity at the time of failure?

**Reply by adamhb** [reply]
Hi @ckoven, 

I found QBOT (specific humidity at the lowest atm level) in the input TPHWL file for the same month that the abort happened. Its 6.161137e-25 kg / kg... seems very low!

But did you mean that I should re-rerun fates and print out the relative humidity to get the value in the exact time step when the model aborts?

Adam

**Reply by ckoven** [reply]
I had been thinking to use Clausius Clapeyron to calculate the relative humidity offline, as I was imagining that somewhere in the process of downscaling and bias correcting, it might have brought the relative humidity close to or above saturation, but it sounds like maybe the opposite is occurring, where humidity (both absolute and relative) are close to zero. Seems like that could be the culprit, though I don't know exactly the mechanism whereby super-low humidities might lead to the error you are encountering.

**Reply by adamhb** [reply]
OK, thanks a lot for drawing my attention to this. Yes, the non-downscaled QBOT values for this same location are in the range of 0.004 to 0.0088 for the same location so it does look like an issue occurred here related to either the downscaling or my own pre-processing.
